src/Domain/Orientation.hpp

```diff
@@ -58,6 +58,9 @@ class Orientation {
   std::array<SegmentId, VolumeDim> mapped(
       const std::array<SegmentId, VolumeDim>& segmentIds) const;
 
+  /// The corresponding Orientation in the frame of the neighbor.
+  Orientation<VolumeDim> inverse() const noexcept;
```

Comment:
I don't really like the term "inverse", I don't know what it means to have the "inverse" of an orientation. Maybe something like `in_neighbor_frame`, `from_neighbor`, or `of_neighbor` (hopefully someone else has an even better suggestion)?

Reply:
I picked `inverse` because when I do `orientation.inverse().inverse()` I get the same orientation back, but I'm open to considering other names.

Reply:
Also because each Orientation corresponds to a rotation of the cube, and the neighbor's orientation corresponds to the inverse of that rotation.